### PR TITLE
Add Google Phishlet (Working for Gmail and Gsuite)

### PR DESCRIPTION
**Pull request by charlesbel**
Hi there,

Here is a working bypass to make work the Google Phishlet. The principe is simple, when the client browser make the "accountlookup" request, we replace the token inside by a new one generated by a headless browser from the official website.
The only problem with this is that the account lookup takes =~ 6-8 seconds.

This code is mainly extracted from this repo : https://github.com/TomAbel/evilginx.botguard
Do not forget to thanks him 😃 

_Note for Kuba_
If you want to approve the pull request, you'll also need to add to the project these 2 packages to make it work :
github.com/go-rod/rod
github.com/go-rod/stealth

### CODE REVIEW

**Comment by TomAbel** [comment]
I don't think this is a good approach. The code only works for one specific site.

**Comment by charlesbel** [comment]
> I don't think this is a good approach. The code only works for one specific site. 

I edited it to only enable on the google phishlet

**Comment by SO-KEi** [comment]
Can you make any bank phishlets? Like the chase, chime, wells Fargo, BOA and the rest. can you? 
email, ***@***.*** if you can make the phishlets.  
 
  On Tue, Sep 7, 2021 at 11:28, ***@***.***> wrote:   



I don't think this is a good approach. The code only works for one specific site.


I edited it to only enable on the google phishlet

**Comment by TomAbel** [comment]
> > I don't think this is a good approach. The code only works for one specific site.
> 
> I edited it to only enable on the google phishlet

Currently, evilginx2 supports specific sites via phishlets. Core code is meant to be used on all sites. I think it should stay that way, but it is up to kgretzky. 

One solution would be to add extra fields to phishlet files, so the go-rod logic can be used on all websites.

**Comment by charlesbel** [comment]
> > > I don't think this is a good approach. The code only works for one specific site.
> > 
> > I edited it to only enable on the google phishlet
> 
> Currently, evilginx2 supports specific sites via phishlets. Core code is meant to be used on all sites. I think it should stay that way, but it is up to kgretzky. 
> 
> One solution would be to add extra fields to phishlet files, so the go-rod logic can be used on all websites.

Yeah you're right. But the core code only launch on the google website, on all the other sites, it does nothing. I think this is the best way to make Google phishlet work.

**Comment by TomAbel** [comment]
You're still pre-loading the browser in [main.go](https://github.com/kgretzky/evilginx2/blob/62d2792d8c86bb3ea50545ec23154e4403ae9434/main.go#L34). This requires installing a chromium browser just to run evilginx2, even if you're not using the google phishlet. For reference, the [chromium-browser](https://packages.ubuntu.com/bionic/amd64/chromium-browser/download) package on ubuntu is 78MB with a full installed size of 267MB.

**Comment by charlesbel** [comment]
> You're still pre-loading the browser in [main.go](https://github.com/kgretzky/evilginx2/blob/62d2792d8c86bb3ea50545ec23154e4403ae9434/main.go#L34). This requires installing a chromium browser just to run evilginx2, even if you're not using the google phishlet. For reference, the [chromium-browser](https://packages.ubuntu.com/bionic/amd64/chromium-browser/download) package on ubuntu is 78MB with a full installed size of 267MB. 
> 
> 

I think we can fix this with a launch argument like --google-bypass, isn't-it ?

**Comment by SO-KEi** [comment]
could anyone of youse get me on in how to work this Evilginx2 on my pc? 
 
 
  On Tue, Sep 7, 2021 at 13:57, ***@***.***> wrote:   



You're still pre-loading the browser in main.go. This requires installing a chromium browser just to run evilginx2, even if you're not using the google phishlet. For reference, the chromium-browser package on ubuntu is 78MB with a full installed size of 267MB.


I think we can fix this with a launch argument like --google-bypass, isn't-it ?

**Comment by ssl-user-en** [comment]
Hi I just checked the new code and I am still getting the something went wrong error. Does it require the server to be in the USA and a TLD to make it work? It always throws the error "Something went wrong"

**Comment by charlesbel** [comment]
> Hi I just checked the new code and I am still getting the something went wrong error. Does it require the server to be in the USA and a TLD to make it work? It always throws the error "Something went wrong"

Did you updated all the files (not just the phishlet) ?

**Comment by ssl-user-en** [comment]
Yes i recompiled the tool from your repo and it still throws me an error of Something Went wrong. I am unable to understand what is blocking the request to go on passwrod page or to get auth tokens.

**Comment by TomAbel** [comment]
> > You're still pre-loading the browser in [main.go](https://github.com/kgretzky/evilginx2/blob/62d2792d8c86bb3ea50545ec23154e4403ae9434/main.go#L34). This requires installing a chromium browser just to run evilginx2, even if you're not using the google phishlet. For reference, the [chromium-browser](https://packages.ubuntu.com/bionic/amd64/chromium-browser/download) package on ubuntu is 78MB with a full installed size of 267MB.
> 
> I think we can fix this with a launch argument like --google-bypass, isn't-it ?

No. It's not just the browser session go-rod tries to install.

@ssl-user-en is most likely experiencing issues since he/she doesn't have a browser installed.

Follow these instructions please: https://github.com/go-rod/go-rod.github.io/blob/master/compatibility.md

TLDR: 
`wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb`

`apt install ./google-chrome-stable_current_amd64.deb`

If you're still experiencing problems I can troubleshoot them, but you need to post evilginx2's debug log here

**Comment by ssl-user-en** [comment]
So i did everything told by @TomAbel and @charlesbel 

Tried the both different verions made by you guys but it still throws me an error Something went wrong.
Could it be because of the vendor of my server or ip or the location of the server or the low level domain I am using?

Here are the logs using the @charlesbel version of evilginx2. I'll redact my email and domain name. Please bear with me.


POST: /signin/v2/identifier
[02:07:06] [dbg] POST body = 
[02:07:06] [dbg] POST: /signin/v2/identifier
[02:07:06] [dbg] accounts.google.com: __Host-GAPS = 1:f9G0Abh85869WkWZfHyQvAKFh92dvQ:6SyqXS8_URhUXiIT
[02:07:07] [dbg] whitelistIP: 103.107.198.108 c99a30373cb828cda571ff0029c6a27663003b3f0a02500a15310d232698ee59
[02:07:07] [dbg] POST: /ServiceLogin
[02:07:07] [dbg] POST body = 
[02:07:07] [dbg] POST: /ServiceLogin
[02:07:07] [dbg] accounts.google.com: __Host-GAPS = 1:CvCozwF7_WzuoslNpyWEy4dIoZ602w:tOu1vK4PJjWvKNc4
[02:07:52] [dbg] whitelistIP: 103.107.198.108 c99a30373cb828cda571ff0029c6a27663003b3f0a02500a15310d232698ee59
[02:07:52] [dbg] POST: /_/bscframe
[02:07:52] [dbg] POST body = 
[02:07:52] [dbg] POST: /_/bscframe
[02:07:52] [dbg] .google.com: NID = 223=bccgcSAUvCsnblgoj_fgO2iITFTBUQ7MmeSz_q1-tgsfLKY9aDKw-Rc1eMSFV4iDyyvewjuFT0Oc95nNnwOLLwdd0cGl1srUAVR-yRjwjx4gJCARAIyEs_IEGDeK7HZu7hbPxwGVNrV_c4gf8Zqgps9loqSaoulEciSOpMMcpUc
[02:07:53] [dbg] whitelistIP: 103.107.198.108 c99a30373cb828cda571ff0029c6a27663003b3f0a02500a15310d232698ee59
[02:07:53] [dbg] POST: /_/kids/signup/eligible
[02:07:53] [dbg] POST body = ifkv=AU9NCcz8KCNPFOMkavhTdF4NFa7MuE8AXkv76XTq9OTKNxFCuwTIVBFHUIDbpLH2KnDi3vCroZNyuA%3D%3D&flowEntry=ServiceLogin&flowName=GlifWebSignIn&continue=https%3A%2F%2Faccounts.abc.xyz%2FManageAccount%3Fnc%3D1&f.req=%5B%22AEThLlyEj_WskoYmy15soXucsND9Zj7-rS85ss72VEANsTjWIgluzODXJyIEnX2jruT1Am8sVpe9lmU8Cn6ezFTECrhi4KilfJsQRh8DKjXThIK_CkcGmqHxRqJi9Jw6VHc36-9fviv_NF3Xof7zpyrLEEB0BrYQ23qoX1e-7aM_LfSMLn9i11XiWvvH03reu3J3EXrpIcvm%22%2C%22S1577933572%3A1631167627868510%22%5D&azt=AFoagUVQg_8dC2V_Y8SpwG9M_M1qGPM3Dw%3A1631167627890&cookiesDisabled=false&deviceinfo=%5Bnull%2Cnull%2Cnull%2C%5B%5D%2Cnull%2C%22DE%22%2Cnull%2Cnull%2Cnull%2C%22GlifWebSignIn%22%2Cnull%2C%5Bnull%2Cnull%2C%5B%5D%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2C%5B%5D%2Cnull%2Cnull%2Cnull%2C%5B%5D%2C%5B%5D%5D%2Cnull%2Cnull%2Cnull%2Cnull%2C0%2Cnull%2Cfalse%2C1%2C%22%22%5D&gmscoreversion=undefined&checkConnection=&checkedDomains=youtube&pstMsg=1&
[02:07:53] [dbg] POST: /_/kids/signup/eligible
[02:07:53] [dbg] POST continue = https://accounts.abc.xyz/ManageAccount?nc=1
[02:07:53] [dbg] POST cookiesDisabled = false
[02:07:53] [dbg] POST deviceinfo = [null,null,null,[],null,"DE",null,null,null,"GlifWebSignIn",null,[null,null,[],null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,[],null,null,null,[],[]],null,null,null,null,0,null,false,1,""]
[02:07:53] [dbg] POST gmscoreversion = undefined
[02:07:53] [dbg] POST checkConnection = 
[02:07:53] [dbg] POST pstMsg = 1
[02:07:53] [dbg] POST ifkv = AU9NCcz8KCNPFOMkavhTdF4NFa7MuE8AXkv76XTq9OTKNxFCuwTIVBFHUIDbpLH2KnDi3vCroZNyuA==
[02:07:53] [dbg] POST flowEntry = ServiceLogin
[02:07:53] [dbg] POST azt = AFoagUVQg_8dC2V_Y8SpwG9M_M1qGPM3Dw:1631167627890
[02:07:53] [dbg] POST checkedDomains = youtube
[02:07:53] [dbg] POST flowName = GlifWebSignIn
[02:07:53] [dbg] POST f.req = ["AEThLlyEj_WskoYmy15soXucsND9Zj7-rS85ss72VEANsTjWIgluzODXJyIEnX2jruT1Am8sVpe9lmU8Cn6ezFTECrhi4KilfJsQRh8DKjXThIK_CkcGmqHxRqJi9Jw6VHc36-9fviv_NF3Xof7zpyrLEEB0BrYQ23qoX1e-7aM_LfSMLn9i11XiWvvH03reu3J3EXrpIcvm","S1577933572:1631167627868510"]
[02:07:53] [dbg] accounts.google.com: __Host-GAPS = 1:lMQe2wctf0wfxKUqouRJ1Od7EP4bcQ:CZCvKRLnorkGLQNS
[02:08:11] [dbg] isWhitelistIP: 5.2.69.50
[02:08:11] [dbg] isWhitelistIP: 91.219.237.21
[02:08:12] [dbg] whitelistIP: 103.107.198.108 c99a30373cb828cda571ff0029c6a27663003b3f0a02500a15310d232698ee59
[02:08:12] [dbg] POST: /_/lookup/accountlookup
[02:08:12] [dbg] POST body = ifkv=AU9NCcz8KCNPFOMkavhTdF4NFa7MuE8AXkv76XTq9OTKNxFCuwTIVBFHUIDbpLH2KnDi3vCroZNyuA%3D%3D&flowEntry=ServiceLogin&flowName=GlifWebSignIn&continue=https%3A%2F%2Faccounts.abc.xyz%2FManageAccount%3Fnc%3D1&f.req=%5B%22username%22%2C%22AEThLlyEj_WskoYmy15soXucsND9Zj7-rS85ss72VEANsTjWIgluzODXJyIEnX2jruT1Am8sVpe9lmU8Cn6ezFTECrhi4KilfJsQRh8DKjXThIK_CkcGmqHxRqJi9Jw6VHc36-9fviv_NF3Xof7zpyrLEEB0BrYQ23qoX1e-7aM_LfSMLn9i11XiWvvH03reu3J3EXrpIcvm%22%2C%5B%5D%2Cnull%2C%22DE%22%2Cnull%2Cnull%2C2%2Cfalse%2Ctrue%2C%5Bnull%2Cnull%2C%5B2%2C1%2Cnull%2C1%2C%22https%3A%2F%2Faccounts.localhostlookalike.xyz%2FServiceLogin%3FflowEntry%3DServiceLogin%26flowName%3DGlifWebSignIn%22%2Cnull%2C%5B%5D%2C4%2C%5B%5D%2C%22GlifWebSignIn%22%2Cnull%2C%5B%5D%5D%2C1%2C%5Bnull%2Cnull%2C%5B%5D%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2C%5B%5D%2Cnull%2Cnull%2Cnull%2C%5B%5D%2C%5B%5D%5D%2Cnull%2Cnull%2Cnull%2Ctrue%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2C%5B%5D%5D%2C%22syfitech.noreply%22%2Cnull%2Cnull%2Cnull%2Ctrue%2Ctrue%2C%5B%5D%5D&bgRequest=%5B%22identifier%22%2C%22%3CPfFq8bICAAZD6Li_Q-iNePP_Wh3odMP0ACkAIwj8RlmaPrNPmK5wAN5R-eTJdKuLw79BXg1oJMp-ucfd8DzZFw8FYMAABBUZTiinAQdWCQQ4GmNuRN6SYrjHpjgZnpch7V9-FdwFp0Zdg2oblhxZKXF5jAVsVkkdiegbjILVUToDfd0-gZWGGu5aNUzZRN_LGCGYy0AOz1bo-VfsJXzdWl1ZOF3qc6YC-ZigSq2TuhewIaQ-os7DVM6ac7gHekWTNIBvqj_nIEwfuSpHsq76KUeL0Uzb5wFVkX0Kdf0Ll-bgb0SjPIQSJccgtFaqp0aLrAMAuBrxR-YZeimJhiBzW7JkfIzCmKhLqz759C7aH5b2MDeFDtlvJN8xWxpKDGFi2fkPIzR5Suc0VmDOY77mZXuFsItmSFkm1ZFIOFN8Irb8Ylf0KBQcsyLne2xyTMqnLcVCPKYGORqtx9ArZbzKOVrR6wVTtH0pCTZZR43Iso923Dio3LeJrb0zL3jd9Yhj3PuaOixvNkTsvUyCspwUekwiLAlpBFieiZnq_x7k-Ifk2rGwNsLv0Ecqs5jrotp7GoXHHFoJB7j9hMkmHiYOioIo_ZEA52KeDwnXkduWZGDGMIiLZTxyzvqv-XOxOXT6WlreMQCyGo1Csm0i3v7LG4SbZiQcArdwCOMG5vGjDuVYgLBMRSlT6cOfIbb7VTIVSaHFMKj_-CMIZh6b-rUfp0FHlw-vTktIe5JhlS0YgAzG-gzFDOfj0B2pfJVzZnIpEkeCadNtLoHOMr_E0Fd-1MfCayYVxePcx0DK4KedtVoSYSK5QpcWM5iHm-ZcWA7eF5Nk1RdaQJqYbiifxdTIRC5Y20KBGtkGT4DbpMDoZPtLPH0Eey6IQ1RrWzNa4vK7xgOuteE-AXXgCXiP9hZtssdq1PJFHTtubjPMgnRybdmvKtVMte71wk13E0SQyVGHoUz-p5MiDQtXzi2Pst-Ag1fAKRRyievKnX6OuNA0bR82vAsL16m5mzXuG9sfIVOjnkTWOBeNpJbcAm_Ha4FxQ4NmlOt8TE__2yvz_Xhbv0C7bsSwWIKr7LBRptOdCzQIT5Qxt7pAPPr4CqYlRWWV98qZEM0ZcV2u5nuy_zXlrGrjfX0ogVsZ7t4AmT_bHt1nu4jvK0Afrbg0f6Np1q5Op1uyOPh5sD8wpO181RvC-rlAq3oefcqNSLOOVQuNMMcqyk-xdaqZDqNmqIbF39XWlrU-w3HW7UaTmzo3beyi-PmUX-xnaZ_9l1S6IdzlTgrGbIlqcKGcBi_fOQ2dGz6kgEJJhni_vt2z5RmsfeowW2z06SB9QzznZny1_RzJ03_uVuONIPOxZnED842j936UWnbbdpXD5c0vYUlCXgKv9NTb5fCHCNppHCgjTEPsFwvDoDHf_ibIm1zEIOSssdEoPP7FoDDuskGgCweWumlvwNgmX3ZTvZ40oRnHRPP8kRnuYbVLC_1PnrERHThawleHdkt2fu39l0cptNs7TgEQgy6kETIld5IcEUlK5-lGGHC8G5WFQBSNPshO45mfAuNhMCnWUN6UJAaEHaCP83m3HkFIRfiYqc6U9tiLDl5WDU6tOf-2bXJMIMySzxIcHXPTye9rQ5i60x_kIXjGHe1xbdBaXmoBZSeyxlYa108weae91Vv_kQnPsza3WjTbysNlrOPavDScnUP8-_AhoRoelsx5jgL3_PCJ-W__OpCp2nfr5dnx6UhDRnFaNxRZfJrQJIEdjdTrM49_CdLmke84ZgDWEwf9cngE-S9tw2ZiI-w_mKBjRABvgVHWmN47jb7gOKuPnoKjq6wDqcdxFbCoOF9U_H_JeGqIzg6Gz-BOQM9q33tD-hkBBEp5lqK10Tl8fwOj0OdmeqTBv96m084H2Jshgw1zkN4fRQpEF0rpF9fL8DKCLyjihoZN9Rv8Yeip-M34B35DdCKnW1jiIZLIdY1tShmgkgfp-c4CUMfWcVzTgRDhRqVR7vg8p1lA64ir7QN758dmLZ5-psnWvKHc1afNaLHbM0wlNDgZyzSYp9o5FZfv1n_rLsNYx5VU6sadDLtdTGgDkIzadtZtaCPo4R3yWVB3rp37U90M9uz_0xwJgIaEiBQslqWHENE94Rl1MYjk45p4NVWJB-8OPI8pD5Iyuu-ZJAjutUmgNhx_tx0H2kDSe4BrFZ8xsyQgdBrxN3nln8KlgXsTzxQRqhziuYJ5qBet8a2us9zDArkn2hqTY-Exd4bno8BV9sDu1kNNHtdUZTCzQqMHInZwZ55LM2nxb7iGE_5GPsYoq7Hcw-VYiYuH6CI3CKGecliV5zPXjtEC5dUyKlXfDmlhC5DPynjK8WZDD30nlLhxLaZptIfWOBHvBu97u4Ez1bUj5-w12NLgxP1pwMOPTYwr5PFFYdD-6VZBpPZb0XZ6Ek9MW5vqAbh1RkMYMCbs1V9bXRvcpJlGtoaVqs5sDGWlac2JxygG3IGb90fURdiKfcIkN0scq_bbUgSGPo2pgaNY1_iVpavYblLz5PElQRFFPe-AlPwydOmqyHKgDfxXwxzioxRhfIhMbdLDSM40-HDkQj1WsDTZRJF4Jz1CxCQQ6Qo-Ruoe-wK0F59bm7j3xEDyuftq6pMhlXXoawiDtPCBaK69AWSemOq2-VQb87a6ohWqXLLgnmBJoqfyn3_Ee3RWu94eYhTarErE-HIq085Z_4fUctwcXNo66bVG_PvBIFOb3iI7VtYxESQJXJj26j01MzNr33z1GvOEcC-ixoUf1eeguK_npVZJ4kZF3eRofKmXd4pUisFFCXMLJHeXTfE8L6GVY2MTch63PhXSM4wO_a5zccj6VlGEQjwE0I3iN4XXgaUdlwOjnZKFsEikMVEbsDJQiuAGUfhuJmRWUIDxDeXQrr7gtk0Lg0tubqO81FVulYkoTuhd1AqGaJOLpIBgslatYdFr_RfVCcxoUTfr1lbz8-tN7nU4ll_9jURQM4JroS3xCQk23ISmPo-B5EnWeEef85J5VESjbxFCQVF141X8kq9P9p1wDUoMoHpMWl2RryQY_1WBJcm1qDnq_uo1JJOuAAruY2QxJ7CeP4QzLyTmk2I7-FiXKQBEBGf5J29SkN0h9EAkd787ivUf3mVLBsp0qS4hcyebcQdHdvx7bes7q7XRGoiUXIeaizqqYddTBE_k4_UM%22%5D&azt=AFoagUVQg_8dC2V_Y8SpwG9M_M1qGPM3Dw%3A1631167627890&cookiesDisabled=false&deviceinfo=%5Bnull%2Cnull%2Cnull%2C%5B%5D%2Cnull%2C%22DE%22%2Cnull%2Cnull%2Cnull%2C%22GlifWebSignIn%22%2Cnull%2C%5Bnull%2Cnull%2C%5B%5D%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2Cnull%2C%5B%5D%2Cnull%2Cnull%2Cnull%2C%5B%5D%2C%5B%5D%5D%2Cnull%2Cnull%2Cnull%2Cnull%2C0%2Cnull%2Cfalse%2C1%2C%22%22%5D&gmscoreversion=undefined&checkConnection=&checkedDomains=youtube&pstMsg=1&
[02:08:12] [dbg] POST: /_/lookup/accountlookup
[02:08:12] [dbg] POST bgRequest = ["identifier","<PfFq8bICAAZD6Li_Q-iNePP_Wh3odMP0ACkAIwj8RlmaPrNPmK5wAN5R-eTJdKuLw79BXg1oJMp-ucfd8DzZFw8FYMAABBUZTiinAQdWCQQ4GmNuRN6SYrjHpjgZnpch7V9-FdwFp0Zdg2oblhxZKXF5jAVsVkkdiegbjILVUToDfd0-gZWGGu5aNUzZRN_LGCGYy0AOz1bo-VfsJXzdWl1ZOF3qc6YC-ZigSq2TuhewIaQ-os7DVM6ac7gHekWTNIBvqj_nIEwfuSpHsq76KUeL0Uzb5wFVkX0Kdf0Ll-bgb0SjPIQSJccgtFaqp0aLrAMAuBrxR-YZeimJhiBzW7JkfIzCmKhLqz759C7aH5b2MDeFDtlvJN8xWxpKDGFi2fkPIzR5Suc0VmDOY77mZXuFsItmSFkm1ZFIOFN8Irb8Ylf0KBQcsyLne2xyTMqnLcVCPKYGORqtx9ArZbzKOVrR6wVTtH0pCTZZR43Iso923Dio3LeJrb0zL3jd9Yhj3PuaOixvNkTsvUyCspwUekwiLAlpBFieiZnq_x7k-Ifk2rGwNsLv0Ecqs5jrotp7GoXHHFoJB7j9hMkmHiYOioIo_ZEA52KeDwnXkduWZGDGMIiLZTxyzvqv-XOxOXT6WlreMQCyGo1Csm0i3v7LG4SbZiQcArdwCOMG5vGjDuVYgLBMRSlT6cOfIbb7VTIVSaHFMKj_-CMIZh6b-rUfp0FHlw-vTktIe5JhlS0YgAzG-gzFDOfj0B2pfJVzZnIpEkeCadNtLoHOMr_E0Fd-1MfCayYVxePcx0DK4KedtVoSYSK5QpcWM5iHm-ZcWA7eF5Nk1RdaQJqYbiifxdTIRC5Y20KBGtkGT4DbpMDoZPtLPH0Eey6IQ1RrWzNa4vK7xgOuteE-AXXgCXiP9hZtssdq1PJFHTtubjPMgnRybdmvKtVMte71wk13E0SQyVGHoUz-p5MiDQtXzi2Pst-Ag1fAKRRyievKnX6OuNA0bR82vAsL16m5mzXuG9sfIVOjnkTWOBeNpJbcAm_Ha4FxQ4NmlOt8TE__2yvz_Xhbv0C7bsSwWIKr7LBRptOdCzQIT5Qxt7pAPPr4CqYlRWWV98qZEM0ZcV2u5nuy_zXlrGrjfX0ogVsZ7t4AmT_bHt1nu4jvK0Afrbg0f6Np1q5Op1uyOPh5sD8wpO181RvC-rlAq3oefcqNSLOOVQuNMMcqyk-xdaqZDqNmqIbF39XWlrU-w3HW7UaTmzo3beyi-PmUX-xnaZ_9l1S6IdzlTgrGbIlqcKGcBi_fOQ2dGz6kgEJJhni_vt2z5RmsfeowW2z06SB9QzznZny1_RzJ03_uVuONIPOxZnED842j936UWnbbdpXD5c0vYUlCXgKv9NTb5fCHCNppHCgjTEPsFwvDoDHf_ibIm1zEIOSssdEoPP7FoDDuskGgCweWumlvwNgmX3ZTvZ40oRnHRPP8kRnuYbVLC_1PnrERHThawleHdkt2fu39l0cptNs7TgEQgy6kETIld5IcEUlK5-lGGHC8G5WFQBSNPshO45mfAuNhMCnWUN6UJAaEHaCP83m3HkFIRfiYqc6U9tiLDl5WDU6tOf-2bXJMIMySzxIcHXPTye9rQ5i60x_kIXjGHe1xbdBaXmoBZSeyxlYa108weae91Vv_kQnPsza3WjTbysNlrOPavDScnUP8-_AhoRoelsx5jgL3_PCJ-W__OpCp2nfr5dnx6UhDRnFaNxRZfJrQJIEdjdTrM49_CdLmke84ZgDWEwf9cngE-S9tw2ZiI-w_mKBjRABvgVHWmN47jb7gOKuPnoKjq6wDqcdxFbCoOF9U_H_JeGqIzg6Gz-BOQM9q33tD-hkBBEp5lqK10Tl8fwOj0OdmeqTBv96m084H2Jshgw1zkN4fRQpEF0rpF9fL8DKCLyjihoZN9Rv8Yeip-M34B35DdCKnW1jiIZLIdY1tShmgkgfp-c4CUMfWcVzTgRDhRqVR7vg8p1lA64ir7QN758dmLZ5-psnWvKHc1afNaLHbM0wlNDgZyzSYp9o5FZfv1n_rLsNYx5VU6sadDLtdTGgDkIzadtZtaCPo4R3yWVB3rp37U90M9uz_0xwJgIaEiBQslqWHENE94Rl1MYjk45p4NVWJB-8OPI8pD5Iyuu-ZJAjutUmgNhx_tx0H2kDSe4BrFZ8xsyQgdBrxN3nln8KlgXsTzxQRqhziuYJ5qBet8a2us9zDArkn2hqTY-Exd4bno8BV9sDu1kNNHtdUZTCzQqMHInZwZ55LM2nxb7iGE_5GPsYoq7Hcw-VYiYuH6CI3CKGecliV5zPXjtEC5dUyKlXfDmlhC5DPynjK8WZDD30nlLhxLaZptIfWOBHvBu97u4Ez1bUj5-w12NLgxP1pwMOPTYwr5PFFYdD-6VZBpPZb0XZ6Ek9MW5vqAbh1RkMYMCbs1V9bXRvcpJlGtoaVqs5sDGWlac2JxygG3IGb90fURdiKfcIkN0scq_bbUgSGPo2pgaNY1_iVpavYblLz5PElQRFFPe-AlPwydOmqyHKgDfxXwxzioxRhfIhMbdLDSM40-HDkQj1WsDTZRJF4Jz1CxCQQ6Qo-Ruoe-wK0F59bm7j3xEDyuftq6pMhlXXoawiDtPCBaK69AWSemOq2-VQb87a6ohWqXLLgnmBJoqfyn3_Ee3RWu94eYhTarErE-HIq085Z_4fUctwcXNo66bVG_PvBIFOb3iI7VtYxESQJXJj26j01MzNr33z1GvOEcC-ixoUf1eeguK_npVZJ4kZF3eRofKmXd4pUisFFCXMLJHeXTfE8L6GVY2MTch63PhXSM4wO_a5zccj6VlGEQjwE0I3iN4XXgaUdlwOjnZKFsEikMVEbsDJQiuAGUfhuJmRWUIDxDeXQrr7gtk0Lg0tubqO81FVulYkoTuhd1AqGaJOLpIBgslatYdFr_RfVCcxoUTfr1lbz8-tN7nU4ll_9jURQM4JroS3xCQk23ISmPo-B5EnWeEef85J5VESjbxFCQVF141X8kq9P9p1wDUoMoHpMWl2RryQY_1WBJcm1qDnq_uo1JJOuAAruY2QxJ7CeP4QzLyTmk2I7-FiXKQBEBGf5J29SkN0h9EAkd787ivUf3mVLBsp0qS4hcyebcQdHdvx7bes7q7XRGoiUXIeaizqqYddTBE_k4_UM"]
[02:08:12] [dbg] POST azt = AFoagUVQg_8dC2V_Y8SpwG9M_M1qGPM3Dw:1631167627890
[02:08:12] [dbg] POST cookiesDisabled = false
[02:08:12] [dbg] POST checkedDomains = youtube
[02:08:12] [dbg] POST pstMsg = 1
[02:08:12] [dbg] POST ifkv = AU9NCcz8KCNPFOMkavhTdF4NFa7MuE8AXkv76XTq9OTKNxFCuwTIVBFHUIDbpLH2KnDi3vCroZNyuA==
[02:08:12] [dbg] POST flowEntry = ServiceLogin
[02:08:12] [dbg] POST f.req = ["username","AEThLlyEj_WskoYmy15soXucsND9Zj7-rS85ss72VEANsTjWIgluzODXJyIEnX2jruT1Am8sVpe9lmU8Cn6ezFTECrhi4KilfJsQRh8DKjXThIK_CkcGmqHxRqJi9Jw6VHc36-9fviv_NF3Xof7zpyrLEEB0BrYQ23qoX1e-7aM_LfSMLn9i11XiWvvH03reu3J3EXrpIcvm",[],null,"DE",null,null,2,false,true,[null,null,[2,1,null,1,"https://accounts.abc.xyz/ServiceLogin?flowEntry=ServiceLogin&flowName=GlifWebSignIn",null,[],4,[],"GlifWebSignIn",null,[]],1,[null,null,[],null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,[],null,null,null,[],[]],null,null,null,true,null,null,null,null,null,null,null,null,[]],"username",null,null,null,true,true,[]]
[02:08:12] [+++] [0] Username: [username]
[02:08:12] [dbg] POST gmscoreversion = undefined
[02:08:12] [dbg] POST checkConnection = 
[02:08:12] [dbg] POST flowName = GlifWebSignIn
[02:08:12] [dbg] POST continue = https://accounts.abc.xyz/ManageAccount?nc=1
[02:08:12] [dbg] POST deviceinfo = [null,null,null,[],null,"DE",null,null,null,"GlifWebSignIn",null,[null,null,[],null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,[],null,null,null,[],[]],null,null,null,null,0,null,false,1,""]
[02:08:12] [dbg] accounts.google.com: __Host-GAPS = 1:lDI5j6jZ-GbxPBFKOKJy-kgWvdTEYQ:RmUZ1UdF3f9ZqdnB

**Comment by ilhan7777** [comment]
Hello there. After entering the e-mail address, I get the following error:

"Couldn’t sign you in
This browser or app may not be secure.
Try using a different browser. "

what would be the reason?

**Comment by ilhan7777** [comment]
https://ibb.co/Cb1x5Gr

**Comment by ilhan7777** [comment]
Hello. 
The script captures the cookie, but there is a problem in the credentials.
When you enter email in Gmail page, Evilginx shows Password: [WZfWSd] (Same for every email.)
So, It does not capture the email address or the correct password.
